samples/notebooks/image-captioning-gcp/README.md

```diff
@@ -0,0 +1,22 @@
+# Image Captioning TF 2.0
+
+## About
+This notebook is an example of how to convert an existing Tensorflow notebook into a Kubeflow pipeline using jupyter notebook.  Specifically, this notebook takes an example tensorflow notebook, [image captioning with attention](https://colab.sandbox.google.com/github/tensorflow/docs/blob/master/site/en/r2/tutorials/text/image_captioning.ipynb), and creates a kubeflow pipeline.  
```

Comment:
Could you add more description of the example. e.g. it is about generating captions for images, etc. maybe with some example prediction images?

Reply:
Added better description + example outputs.